rollup-plugin-docz-starter.ts

```diff
@@ -20,7 +20,7 @@ function doczStarter() {
             const docz = spawn('npx', ['docz', 'dev']);
             started = true;
 
-            docz.on('close', code => {
+            docz.on('close', (code) => {
```

Comment:
I know this is probably going to bug some people out. Prettier has changed their default setting for whether or not to use parenthesis in arrow functions with a single argument, and I think their reasoning is pretty good to be honest ([link](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev)).

No one will have to change how they write these, since the linter will autofix them anyway.